qa/tasks/cephfs/test_scrub_checks.py

```diff
@@ -30,22 +30,48 @@ def _resume_scrub(self, expected):
         self.assertEqual(res['return_code'], expected)
     def _get_scrub_status(self):
         return self.fs.rank_tell(["scrub", "status"])
-    def _check_task_status(self, expected_status):
-        task_status = self.fs.get_task_status("scrub status")
-        active = self.fs.get_active_names()
-        log.debug("current active={0}".format(active))
-        self.assertTrue(task_status[active[0]].startswith(expected_status))
+    def _check_task_status(self, expected_status, timo=120):
+        check = 0
+        while check < timo:
+            active = self.fs.get_active_names()
+            log.debug("current active={0}".format(active))
+            task_status = self.fs.get_task_status("scrub status")
+            try:
+                if task_status[active[0]].startswith(expected_status):
+                    return True
+            except KeyError:
+                pass
+            check += 1
+            time.sleep(1)
+        return False
+
+    def _check_task_status_na(self, timo=120):
```

Comment:
`na` is?

Reply:
"not available" -- will make that clear.